btrack/btypes.py

```diff
@@ -511,7 +501,7 @@ def to_array(
 
         tmp_track = np.concatenate(tmp_track, axis=-1)
         assert tmp_track.shape[0] == len(self)
-        assert tmp_track.ndim == 2
+        assert tmp_track.ndim == TWO_DIM
```

Comment:
We have a enum for these already:
```suggestion
        assert tmp_track.ndim == constants.Dimensionality.TWO
```

Reply:
oh nice, there are a few more places `TWO_DIM` is used so I'll go through and change to `constants.Dimensionality.TWO`